CHANGES.md

```diff
@@ -19,6 +19,7 @@ To be released.
 
  -  Added `TurnClient.BindProxies()` method. [[#756], [#868]]
  -  Added `ActionEvaluation.Exception` property.  [[#860], [[#875]]]
+ -  Added `render` optional parameter to `BlockChain<T>()` constructor.  [[#883]]
```

Comment:
`BlockChain<T>()` and `BlockChain<T>(render=true)` are different method in IL. so users should recompile their binary with `BlockChain<T>(render=true)` once again. should we mention it? @planetarium/libplanet

Reply:
Is it better we move it to Backward-incompatible API changes?

Reply:
Moved the changelog to Backward-incompatible API changes.